### PR TITLE
Fixes #2383 :  Add a toast message when user clicks on "delete local search history" 

### LINKED ISSUE #2383: Add a toast message when user clicks on "delete local search history"
**Checklist**
- [x] Bug Report - The issue you are reporting might be there already. Please search on the [issues tracker](https://github.com/openfoodfacts/openfoodfacts-androidapp/issues) before creating or reporting an issue.
- [x] Before reporting, please ensure (if you can) that you have tested it with the latest (dev) version of the software.
- [x] Don't forget to tag your issue, it makes it easier to discover.
**Summary:** 

when user clicks on delete local history in preferences ..a toast message must appear.

**Steps to reproduce:** 

1.Go to preferences.
2.Click on "delete local search history"
3.See the error.

**Device and Android version:** 

Asus 3s max.
**Screenshots:** 
scan

I want to work on this issue

### PR DESCRIPTION
**Pull request by mustaqmustu**
## Description
Updated preferenceFragment.java and srings.xml

## Related issues and discussion
#2383 

Screenshots:
deleted
 
 ## Checklist
 
Make sure you've done all the following (You can delete the checklist before submitting)
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 - [x] Code is well documented
 - [x] Include unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [x] Code passes Travis builds in your branch
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -626,7 +626,7 @@
 
     <string name="scan_first_product">Scan First Product</string>
 
-
+    <string name="preference_delete_search_history">Successfully deleted</string>
```
Comment (Karljoones) [inline app/src/main/res/values/strings.xml]: Can you make this more specific? "Successfully deleted history" or something?

Comment (mustaqmustu) [reply app/src/main/res/values/strings.xml]: yep.Thanks for the review.Changed it .Check it out now.

Comment (mustaqmustu) [reply app/src/main/res/values/strings.xml]: @Karljoones I didn't made any extra changes ..but build is failing.

Comment (Karljoones) [reply app/src/main/res/values/strings.xml]: It's possible that another merge has made it fail

**Comment by mustaqmustu** [comment]
@Karljoones can u merge it now :)

**Comment by Karljoones** [comment]
Merged, thanks @mustaqmustu

**Pull request merged**